src/index.ts

```diff
@@ -11,16 +11,19 @@ export * from './validations'
 export const createValidator = (
   externalCalls: ExternalCalls,
   components?: Pick<ContentValidatorComponents, 'logs'>
-): Validator => ({
-  validate: async (deployment) => {
-    const logs = components?.logs.getLogger('ContentValidator')
-    for (const validation of validations) {
-      const result = await validation.validate({ deployment, externalCalls }, logs)
-      if (!result.ok) {
-        logs?.debug(`Validation failed:\n${result.errors?.join('\n')}`)
-        return result
+): Validator => {
+  const logs = components?.logs.getLogger('ContentValidator')
+
+  return {
+    validate: async (deployment) => {
+      for (const validation of validations) {
+        const result = await validation.validate({ deployment, externalCalls }, logs)
+        if (!result.ok) {
+          logs?.debug(`Validation failed:\n${result.errors?.join('\n')}`)
```

Comment:
is there a reason to continue having loggers as an optional thing?

Reply:
I though it was a good idea to set this component as optional, so other libraries can use it without providing a logger: 

https://github.com/decentraland/content-validator/blob/main/src/index.ts#L13

maybe we can revisit the idea of instantiate a default logger component there